lighthouse-core/test/lib/traces/metrics-evts-test.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const Metrics = require('../../../lib/traces/metrics-evts');
+const assetSaver = require('../../../lib/asset-saver');
+const assert = require('assert');
+
+const dbwTrace = require('../../fixtures/traces/dbw_tester.json');
+const dbwResults = require('../../fixtures/dbw_tester-perf-results.json');
```

Comment:
was hoping you'd ask that. :) In this case it's pretty tricky because we need mock results and mock trace from the exact same run. Currently we have no fixtures that satisfy that.

I attempted to update one of our `sample.json`'s (yes we have two) but after 25 minutes i concluded the rabbit hole was a bit too deep for me. :/